tooling/metamodel-generator/src/main/java/org/hibernate/jpamodelgen/validation/MockSessionFactory.java

```diff
@@ -177,7 +177,7 @@ public Class<?> classForName(String className) {
 		metamodel = new MockMappingMetamodelImpl();
 
 		bootModel = new MetadataImpl(
-				UUID.randomUUID(),
+				SafeRandomUUIDGenerator.safeRandomUUID(),
```

Comment:
This change seems unnecessary.

Reply:
Not sure if it's absolutely necessary, but is it a problem?

Reply:
Let's not change code that doesn't have to change. Please revert this one change.